src/articulation.js

```diff
@@ -12,12 +12,95 @@
 import { Vex } from './vex';
 import { Flow } from './tables';
 import { Modifier } from './modifier';
-import { StaveNote } from './stavenote';
 import { Glyph } from './glyph';
+import { Stem } from './stem';
 
 // To enable logging for this class. Set `Vex.Flow.Articulation.DEBUG` to `true`.
 function L(...args) { if (Articulation.DEBUG) Vex.L('Vex.Flow.Articulation', args); }
 
+const { ABOVE, BELOW } = Modifier.Position;
+
+const mathToIncrement =
```

Comment:
This function is just used once in the code (AFAICT) with a constant mathFn. It makes `snapLineToStaff` harder to read, so doesn't seem worth it.